manifest.yaml

```diff
@@ -1,12 +1,79 @@
 compat: v0
 id: c-lightning
-version: 0.10.0
+version: 0.10.1
```

Comment:
keep the native semver - just update with our internal last digit ie. 0.10.0.1

Reply:
This update would potentially pull in 0.10.1, which is the latest release tag from upstream

Reply:
oh gotcha I did not realize - then 0.10.1 is righteous